lib/Sender/SenderApiClient.php

```diff
@@ -99,12 +99,14 @@ public function checkApiKey()
             $status = curl_getinfo($ch, CURLINFO_HTTP_CODE);
             curl_close($ch);
 
-            if($status === 200){
+            if ($status === 401) {
```

Comment:
as sakyciau return true tik tuo atveju jei 200, kitaip false

Reply:
@uksarkar what do you think of these change? Was thinking that only return false and then this would disconnect when we get a response from here in 401.

Reply:
I think we should return `$status === 200`, if somewhy it's not `200` that indicates we have a problem somewhere else, which could break some other parts as well like export customers.

Reply:
yeah but what would happen if the error is on our side like 500 error, or something from an endpoint changes and gives 422, this would make the plugin to remove the api key as we only allow 200 responses.